### DISCUSSION TITLE
HLS playback in Android/IOS mobile

### DISCUSSION THREAD

**Discussion by uzername123** [bluenviron/mediamtx#921] [Questions]
Hi all. have an 0.18 version Linux (x86_64 and aarch64) , in both versions stream looks ok as rtmp, rtsp , but with HLS have somwe strange things - stream looks absolutely ok in desktop browser and sometime looks like broken stream in mobile Android  http browser (chrome and Samsung) and IOS mobile - like missed reference frame - like moving objects reveals existing picture details over black screen parts or like mobile player displaying only changing details.

**Comment by aler9** [comment]
open an issue and provide instructions on how to replicate.
